### PR TITLE
Checked methods in StoreController, corrected errors

### PR DESCRIPTION
**Pull request by grigveronika**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/exception/ErrorCode.java [hunk 1]
```diff
@@ -7,7 +7,8 @@
 public enum ErrorCode {
     NOT_FOUND_STORE("Store not found."),
     NOT_FOUND_PRODUCT("Product not found."),
-    NOT_FOUND_PROFILE("Profile not found.");
+    NOT_FOUND_PROFILE("Profile not found."),
+    STORE_IS_NULL("Store is null");
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/exception/ErrorCode.java:11]: Store does not exist

File: src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java [hunk 2]
```diff
@@ -72,14 +75,13 @@ public static StoreDTO fromStoreToStoreDTO(Store store) {
...
         }
-        //    target.setFileType(fileTypeConverter.toDto(source.getFileType()));
         return StoreDTO.builder()
                 .id(store.getId())
                 .storeName(store.getStoreName())
                 .description(store.getDescription())
                 .address(store.getAddress())
                 .profit(store.getProfit())
-                //   .profileDTO()
+                .profileDTO(ProfileServiceImpl.convertEntityToDTO(store.getProfile()))
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java:84]: entity? what entity?

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java (hunk 2 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/service/impl/StoreServiceImpl.java:84]: Masha will fix it now, it’s in her method

File: src/main/java/internship/mbicycle/storify/exception/ErrorCode.java (hunk 1 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/exception/ErrorCode.java:11]: deleted STORE_IS_NULL("Store is null"), as long as I don’t need him

**Pull request merged**